time_execution/backends/threaded.py

```diff
@@ -74,12 +75,17 @@ def send_metrics():
                 logger.warning('%r write failure %r', self.backend, exc)
 
         while self.has_work():
+            if self.thread:
```

Comment:
In some cases the timeexecution thread causes short lived processes to segfault.